FUEL/household.py

```diff
@@ -75,20 +76,18 @@ def reformat_dataframe(dataframe):
 
         dataframe = dataframe.apply(lambda x: np.float64(x) if x.name != 'timestamp' else x)
 
-        if type(dataframe['timestamp']) is str:
-            dataframe['timestamp'] = pd.to_datetime(dataframe['timestamp'], format='%m/%d/%Y %H:%M')
+        dataframe['timestamp'] = dataframe['timestamp'].astype('datetime64[ns]')
 
         return dataframe
 
     data = pd.read_csv(datafile_path, header=None)
     df_stoves, stoves, fuels = stove_info(data)
-
     return df_stoves, stoves, fuels
```

Comment:
I like that you added this functionality above the class... it would possibly would be a bit cleaner to store this functionality in a different file (reformat_files.py or something). Up to you though